pom.xml

```diff
@@ -113,7 +113,7 @@
         (https://nvd.nist.gov/vuln/detail/CVE-2020-9488#vulnCurrentDescriptionTitle), the SMTP appender is
          excluded from the build and not packaged into the agent artifacts -->
         <version.log4j>2.12.4</version.log4j>
-        <version.log4j2-ecs-layout>1.4.0</version.log4j2-ecs-layout>
+        <version.ecs.logging>1.4.1-SNAPSHOT</version.ecs.logging>
```

Comment:
[for reviewer] using a single property for all `ecs-logging-java` artifacts, will require to be updated with a non-snapshot once ecs-logging-java is released.